app/helpers/application_helper.rb

```diff
@@ -24,6 +24,14 @@ def wordpress_path(path = '')
     Settings.wordpress.url.chomp('/') + '/' + path.sub(/^\/+/) { $1 }
   end
 
+  ##
+  # Returns frontend path with correctly joining '/'
+  # @param path String (with or without leading slash)
+  # @return String
+  def api_path(path = '')
+    Settings.api.url.chomp('/') + '/' + path.sub(/^\/+/) { $1 }
```

Comment:
@anarchivist I made a change to ensure that the links to the item in the frontend and the API are both `http` URIs.  I also changed the settings file and the specs to reflect the new, protocol-neutral way we are defining paths to other frontend websites: https://github.com/dpla/primary-source-sets/pull/153/commits/9000ce5e079bdbd9a2d9748480eeb2ce13217046